workflow-templates/cppcheck.yml

```diff
@@ -1,7 +1,7 @@
 name: cppcheck
 on:
   pull_request_target:
-    types: [opened, reopened, synchronize, closed]
+    types: [opened, reopened, synchronize]
```

Comment:
You can delete this line when types only include `[opened, reopened, synchronize]`, they are default value. 

> By default, a workflow only runs when a pull_request_target's activity type is opened, synchronize, or reopened

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target